specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/examples/BillingProfileRoleAssignmentList.json

```diff
@@ -0,0 +1,85 @@
+{
+    "parameters": {
+        "api-version": "2018-08-01-preview",
+        "billingAccountName": "2000100",
+        "billingProfileName": "{billingProfileName}"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [
+                    {
+                        "id": "10000000-aaaa-bbbb-cccc-100000000000_6fd330f6-7d26-4aff-b9cf-7bd699f965b9",
+                        "name": "10000000-aaaa-bbbb-cccc-100000000000_6fd330f6-7d26-4aff-b9cf-7bd699f965b9",
+                        "properties": {
+                            "createdOn": "2018-06-21T21:34:12.2363515+00:00",
+                            "createdByPrincipalTenantId": "7ca289b9-c32d-4f01-8566-7ff93261d76f",
+                            "createdByPrincipalId": "",
+                            "name": "10000000-aaaa-bbbb-cccc-100000000000_6fd330f6-7d26-4aff-b9cf-7bd699f965b9",
+                            "principalId": "6fd330f6-7d26-4aff-b9cf-7bd699f965b9",
+                            "roleDefinitionId": "/providers/Microsoft.Billing/billingAccounts/ee4241f3-de74-5631-3e6b-5df2c3d66671/billingProfile/123/providers/Microsoft.Billing/roleDefinitions/10000000-aaaa-bbbb-cccc-100000000000",
```

Comment:
Additional properties not allowed: roleDefinitionId

Same else where in this example